src/main/java/com/google/events/cloud/audit/v1/DestinationAttributes.java

```diff
@@ -47,8 +47,8 @@
     /**
      * The network port of the peer.
      */
-    public NumResponseItems getPort() { return port; }
-    public void setPort(NumResponseItems value) { this.port = value; }
+    public String getPort() { return port; }
```

Comment:
I would really like us to be able to represent this as a `Long`. Yes, it's a string in the JSON itself, but a string that should always represent a 64-bit integer. As a user, I would *definitely* expect it to be a numeric type of some description. We don't need to fix this in this PR, but I really think we need a concerted look at the accuracy and usefulness of the results in all languages. (I think we need to look at numbers, bytes fields, timestamps, durations, Struct, and the various wrapper types.)

Reply:
I fixed this issue in the last commit.